sql_upgrade.php

```diff
@@ -202,8 +203,36 @@ function progressStatus(msg = '') {
     let eventList = document.getElementById('status-message');
     let progressEl = document.getElementById('progress');
 
-    progressEl.style.width = processProgress + "%";
-    progressEl.innerHTML = processProgress + "%" + " v" + currentVersion;
+    if (currentVersion == "UUID") {
```

Comment:
@sjpadgett , thought you'd like this status for the uuid (it basically just increments slower and slower as time moves on making it look like it is progressing without having any idea about the actual time it will take; it never gets to 99%, but when it's done it gets set to 100% and moves on :) ).
uuid creation is way faster now with @adunsulag 's changes (for example, just took a minute or so for couple hundred thousand uuids on my rough testing in upgrade with 10000 random patients)